### ISSUE TITLE
Feature request: Sort tabs

### ISSUE THREAD

**Issue by patrik-krajcovic2** [onemen/TabMixPlus#201]
-Sort tabs by URL ascending
-Sort tabs by URL descending
-Sort tabs by title ascending
-Sort tabs by title descending

**Comment by Salvora** [comment]
+1 for this. There is already an extension for this but it is nice to integrate this functionality to TBM. https://addons.mozilla.org/en-US/firefox/addon/sort-tabs-advanced/

**Comment by onemen** [comment]
![sort](https://github.com/user-attachments/assets/f0a31742-36f4-423a-9c9e-517087caecd7)

This is what we have.
It sort the tabs in the list

**Comment by Salvora** [comment]
Yep, I know that one and it is useful but using that sort "option" does not get reflected on the actual tabs. It just sorts that specific list.
